spec/features/reservations_spec.rb

```diff
@@ -176,6 +176,64 @@
     end
   end
 
+  context 'banned equipment processing' do
+    after(:each) do
+      @user.role = 'patron'
+      @user.save
+    end
+    shared_examples 'can handle banned user reservation transactions' do
+      it 'checks in successfully' do
+        # check in
+        @checked_out_res = FactoryGirl.create :checked_out_reservation,
+                                              reserver: @user,
+                                              equipment_model: @eq_model
+        @user.role = 'banned'
```

Comment:
You can swap this and the next line with `@user.update_attributes(role: banned)` to keep things short and sweet :smile:.